.github/workflows/ci.yml

```
@@ -161,13 +156,17 @@ jobs:
         run: |
           rustup install nightly
           rustup +nightly component add clippy
+          rustup +nightly component add rustfmt
           cargo install cargo-dylint dylint-link || true
           cargo install cargo-license            || true
           cargo install cargo-sort               || true
           cargo install cargo-supply-chain       || true
           cargo install cargo-unmaintained       || true
           cargo install group-runner             || true
 
+      - name: Install actionlint
+        run: go install github.com/rhysd/actionlint/cmd/actionlint@latest
```

```suggestion
          go install github.com/rhysd/actionlint/cmd/actionlint@latest
```
Sorry, I was unclear. You can put the `go install ...` after the last `cargo install ...` in the "Install tools" step.

Please also don't forget to use `std::env::home_dir` instead of `home::home_dir` (and remove `home`).